common/djangoapps/track/shim.py

```diff
@@ -113,6 +113,9 @@ def __call__(self, event):
             return
```

Comment:
nit: On line 106 above (while you're in the code), either fix the misspelling in the comment and update with a 'why', or remove altogether.